### ISSUE TITLE
mega-get -m duplicates downloaded files

### ISSUE THREAD

**Issue by moledzki** [meganz/MEGAcmd#586]
Hi,

According to mega-get help -m option should merge contents of existing directory with downloaded one.
But instead of this duplicate files (with sequential number in parentheses) are created in existing directory.

**Comment by phoenix1887** [comment]
Indeed, I don't understand the difference between not using the m flag and using the m flag. Can developers explain and add more content / examples to the help to explain this to users? It's not clear what the behavior is like and how it differs with and without using the m flag.

Also, what if we want it to download a directory and overwrite local existing files if they are different on the mega server instead of creating a new file with a number appended to the file name? How do we achieve that?

**Comment by CanOfSocks** [comment]
This is still occurring despite the help menu stating that it merges the folders, preserving existing, when instead it downloads duplicates
